test/labeling/test_utils.py

```diff
@@ -18,3 +18,16 @@ def test_filter_unlabeled_dataframe(self) -> None:
         np.testing.assert_array_almost_equal(
             y_filtered, np.array([[0.25, 0.25, 0.25, 0.25], [0.2, 0.3, 0.5, 0.0]])
         )
+
+    def test_create_cv_data(self) -> None:
+        L_train = np.array([[1, 1, 0], [-1, -1, 1], [0, 0, 1], [1, 1, 0]])
+        L_dev = np.array([[-1, 1, 1], [0, -1, 0]])
+        Y_dev = np.array([-1, 1])
```

Comment:
is -1 a valid value here?

Reply:
and this doesn't error because -1s are ignored in gold labels with the Scorer() object!